Plugins/AdminSetup/Program.cs

```diff
@@ -51,7 +45,10 @@ static int Main(string[] args)
             string errorMessage = null;
             try
             {
-                Paths.InitializeRhetosServerRootPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\.."));
+                var configurationProvider = new Rhetos.ConfigurationBuilder().
+                    AddRhetosAppConfiguration(GetRhetosRootFolder()).Build();
+                LegacyUtilities.Initialize(configurationProvider);

```

Comment:
Intention is not clear to me here. Probably LegacyUtilities are needed even if try..catch fails? 

However, this configuration is redundant and slightly incompatible with the one created at the start of try...catch block. Maybe create container at the top code level to clearly display when it happens as it also initializes LegacyUtilities implicitly. 

If container creation fails itself, I'm not sure if the code is expected to continue at all?

Reply:
This allows call to SetUpAdminAccount to work even if CreateAdminUserAndPermissions failed. It is a strange feature, but we'll leave it for now.
We have modified the code and added comment to make the intention more clear.